src/pages/guides/Authorization to File Upload/index.md

```diff
@@ -0,0 +1,140 @@
+
+## Add an API
+
+Using APIs allows your application to make calls to Adobe services by means of a REST API. APIs can be connected to your app in multiple ways, depending on the API or the type of app that you are building. To learn more about each of these authentication methods or API connections, read the [authentication documentation](https://developer.adobe.com/developer-console/docs/guides/authentication/).
+
+* [Add an API using OAuth Server-to-Server credential](https://developer.adobe.com/developer-console/docs/guides/services/services-add-api-oauth-s2s)
+* [Add an API using OAuth User authentication credential](https://developer.adobe.com/developer-console/docs/guides/services/services-add-api-oauth-user-authentication)
+* [Add an API using API Key authentication](https://developer.adobe.com/developer-console/docs/guides/services/services-add-api-key)
+
+## Upload a New File
+
+There are two ways to upload a new file: `Create File (local upload)` and `Create File (remote upload)`.
+
+**Local Upload**
+
+To create a file through local upload, select the **Create File (local upload)** endpoint. The request body requires the file name and its file size.
+
+![Request body](../image_12.png)
+
+* If the request is successful, a placeholder file resource is created without any content. Depending on the file size, the response body will include one or more `upload_urls`. See [Multi-part Upload](#multi-part-upload) for next steps.
+
+![Upload url](../image_13.png)
+
+**Remote Upload**
+
+To create a file through remote upload, select the **Create File (remote upload)** endpoint. The request body requires the file name and its source url.
+
+![Remote upload](../image_14.png)
+
+* If the request is successful, the file appears in the UI.
+
+![Request](../image_15.png)
+
+![Request 2](../image_16.png)
+
+## Multi-part Upload  
+
+After creating a placeholder file through the **Create File (local upload)** endpoint, you will have one or more upload urls. These upload urls are required to make upload requests. It may be useful to compose a shell script that splits up the source file into chunks and issues the same number of subsequent requests.
+
+> **NOTE:** These are important details to keep in mind when sending the subsequent upload requests.
+
+> * The HTTP request method must be `PUT`.
+> * The `x-amz-acl` header must be included and be set to private.
+> * The `Content-Type` header must match the `media_type` specified in the original **Create File (local upload)** request. This is true even when uploading the file as separate parts.
+
+In the sample Python script below, we're passing in one upload url in the `upload_urls` parameter, though as noted above, you may have more than one provided to you based on the file size set in the request body for **Create File (local upload).**
+
+``` python
+import requests
+import math
+from typing import List
+from tqdm import tqdm  # For progress bar
+
```

Comment:
May be better here to indicate `chunk_size` is optional and defaults to `none`. @Sahedeva what do you think? 
`def upload_file_in_chunks(file_path: str, upload_urls: list[str], chunk_size: int | None = None) -> bool:`